style/main.css

```diff
@@ -0,0 +1,197 @@
+html {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    font-family: 'Roboto', Helvetica, Arial, Verdana, sans-serif;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+.flex {
+    display: flex;
+}
+
+.col {
+    flex-direction: column;
+}
+
+body {
+    max-width: 1440px;
+    margin: auto;
+}
+
+header {
+    margin: 0;
+    background:url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat; 
+    background-size: 1440px 1315px; 
+    height: 550px;
+    align-items: center;
+    color: #fff;
+    justify-content: center;
+}
+
+#main-nav {
+    text-align: center;
+    position: absolute;
+    top: 2em;
+    width: 980px;
+    height: 36px;
+    border-radius: 4px;
+    background: linear-gradient( #686868, #727272, #737373, #717171, #717171, #6e6e6e, #6c6c6c, #6a6a6a, #676767, #666666, #646464, #626262, #616161, #606060, #5f5f5f, #5f5f5f, #5e5e5e, #5e5e5e, #707070, #717171, #717171, #727272, #747474, #767676, #777777, #797979, #7b7b7b, #7d7d7d, #808080, #828282, #848484, #878787, #8d8d8d, #8d8d8d);
+    font-size: 0.85em;
+}
+
+.nav-link {
+    width: 106px;
+    cursor: pointer;
+    border-right: 1px solid #535353;
+    position: relative;
+}
+
+.overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    background-color: #313131;
+    opacity: 0;
+}
+
+.nav-link:hover .overlay {
+    opacity: 0.7;
+    align-items: center;
+    justify-content: center;
+}
+
+.nav-link:first-child:hover .overlay  {
+    border-radius: 4px 0 0 4px;
+}
+
+
+.nav-item {
+    height: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.fa-apple {
+    font-size: 22px;
+}
+
+#search-div {
+    width: 119px; 
+    position: relative;
+}
+
+input:focus {
+    outline: none;
+}
+
+#search-div .fa-search {
+    position: absolute;
+    font-size: 12px;
+    top: 34%;
+    left: 13%;
+    color: #fff;
+}
+
+#search {
+    width: 106px;
+    height: 20px;
+    border-radius: 10px;
+    text-indent: 1.8em;
+    font-size: 12px;
+    background: linear-gradient(#686868, #727272, #737373, #717171, #717171, #6e6e6e, #6c6c6c, #6a6a6a, #676767, #666666, #646464, #626262, #616161, #606060, #5f5f5f, #5f5f5f, #5e5e5e, #5e5e5e, #707070, #717171, #717171, #727272, #747474, #767676, #777777, #797979, #7b7b7b, #7d7d7d, #808080, #828282, #848484, #878787, #8d8d8d, #8d8d8d);
+    border: none;
+    box-shadow: 1px -2px #3b3b3b;
+}
+
+#hero-wrapper {
+    align-items: center;
+    justify-content: flex-start;
+    color: white;
+    height: 285px;
+}
+
+.hero-text {
+    margin: 0 15%;
+    text-align: center;
+    font-weight: 100;
+}
+
+#hero-title {
+    font-size: 64px;
+    margin-top: 0.7em;
+}
+
+#hero-subtitle {
+    margin-bottom: 0.5em;
+    font-size: 30px;
+    margin-top: 0.5em;
+    margin-bottom: 1.1em;
```

Comment:
You are overwriting this property (Check line 133)

Reply:
Removed line 133